src/CliMenu.php

```diff
@@ -319,6 +319,27 @@ protected function draw() : void
         $frame = new Frame;
 
         $frame->newLine(2);
+        
+        $borderRow = [
```

Comment:
Maybe this could be stored in the style so that it doesn't need to be recomputed each time but just when one of the values change, I'm not sure.